django_celery_token_bucket/bucket.py

```diff
@@ -46,7 +54,18 @@ def _get_or_create_schedule(self):
     def create_periodic_task(self):
         from django_celery_beat.models import PeriodicTask
 
-        crontabschedule = self._get_or_create_schedule()
+        defaults = dict(
+            queue=self.token_refill_queue,
+            kwargs=json.dumps(dict(name=self.name)),
+            task="django_celery_token_bucket.tasks.token_bucket_refill",
+            crontab=None,
+            interval=None
+        )
+
```

Comment:
TBD:
Should we fail if both of `schedule` and `interval_in_seconds` are set or accept it? It might be a valid use case to refill every x seconds and on a specific cron schedule.
What we definitely should do is raise an exception if none of them is provided.

Reply:
> Hey @Vidski and thanks for your contribution. Wouldn't it be possible to just stack the decorators?
> 
> ```python
> @celery_app.task(bind=True)
> @rate_limit(token_bucket="bucket_one", countdown=300)
> @rate_limit(token_bucket="bucket_two", countdown=300)
> def my_tasK(self, *args, **kwargs):
>     return
> ```

This is the way 👍

Reply:
Good question. I added validation for both variables and only accepting one for now

Reply:
according to the source code, you can only set one type of interval so for now this implementation is our best bet. 
https://github.com/celery/django-celery-beat/blob/main/django_celery_beat/models.py